data/config-schema/src/main/resources/net/corda/schema/configuration/rest/1.0/corda.rest.json

```diff
@@ -2,46 +2,46 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "$id": "https://corda.r3.com/net/corda/schema/configuration/rest/1.0/corda.rest.json",
   "title": "Corda REST Configuration Schema",
-  "description": "Configuration schema for the REST subsection.",
+  "description": "Configuration schema for the REST section.",
   "type": "object",
   "default": {},
   "properties": {
     "address": {
-      "description": "",
+      "description": "The address at which the REST API is exposed.",
       "type": "string",
       "default": "0.0.0.0:8888"
     },
     "context": {
-      "description": "",
+      "description": "Settings for the context of the API.",
```

Comment:
My descriptions in this section are obviously just guesses. Is there more we can say? Why would someone want to change these?